### PR TITLE
add bootstrap, style login-page

### PR DESCRIPTION
**Pull request by ghost**
@ricardo-quinones

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -13,3 +13,34 @@
...
+}
+
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  /* Set the fixed height of the footer here */
+  height: 60px;
+  background-color: #f5f5f5;
+}
```
Comment (ricardo-quinones) [inline app/assets/stylesheets/application.css.scss]: New line plz. You can also have Sublime Text do this for automatically by adding

```json
"ensure_newline_at_eof_on_save": true
```
to your User preferences file.

File: app/assets/stylesheets/shared.css.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+// Header
+.navbar.navbar-default {
+  background-color: #450b6b;
+  border-color: none;
+  height: 60px;
+  margin-top: auto;
+}
+
+.navbar-default .navbar-nav > li > a {
+  font-family: QuattrocentoSans-Bold, sans-serif;
```
Comment (ricardo-quinones) [inline app/assets/stylesheets/shared.css.scss]: We can declare this on the body and have everything inherit.

**Comment by ricardo-quinones** [comment]
As general comment, we should be trying to organize the CSS as components: links, buttons, typography, cards, etc. That makes everything more reusable and it should remove the need for `!important`declarations. Also, as much as possible, try to style via only classes as opposed to ids.

File: app/assets/stylesheets/components/login_flow.css.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  margin-bottom: 65.5px;
+  width: 278px;
+  height: 52px;
+  opacity: 0.7;
+  font-size: 24px;
+  color: $text-light;
+  margin-top: 29.4
+}
+
+img.icon {
```
Comment (signalplus-dev) [inline app/assets/stylesheets/components/login_flow.css.scss]: You shouldn't need `img.icon`. `.icon` should suffice.

File: app/assets/stylesheets/components/login_flow.css.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  margin-right: 12.5px;
+}
+
+a.twitter-create-link {
```
Comment (signalplus-dev) [inline app/assets/stylesheets/components/login_flow.css.scss]: Same here. Shouldn't need to have the tag and the class.

File: app/assets/stylesheets/elements/_buttons.css.scss [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    width: 108px;
+    height: 20px;
+    font-family: $default-font-bold;
+    font-size: 15px;
+  }
+}
+
+.nav-pills > li.active > a,
+.nav-pills > li.active > a:hover,
+.nav-pills > li.active > a:focus {
```
Comment (signalplus-dev) [inline app/assets/stylesheets/elements/_buttons.css.scss]: A better way is to write it like

``` sass
.nav-pills > li.active > a {
  &,
  &:active,
  &:focus {
    // etc.
  }
}
```

Comment (signalplus-dev) [inline app/assets/stylesheets/elements/_buttons.css.scss]: If we could get by with just a single class declaration on the `a` tag that would be preferred. I don't know if you are fighting CSS specificity here or not but it's always better to have flatter CSS class declarations that way you don't always have to fight specificity.

File: app/assets/stylesheets/elements/_buttons.css.scss [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    color: $text-light;
+    background-color: $light-gray;
+    width: 150px;
+    height: 49px;
+}
+
+// Login Button
+a.btn-login {
```
Comment (signalplus-dev) [inline app/assets/stylesheets/elements/_buttons.css.scss]: No `a`.

File: app/assets/stylesheets/elements/_buttons.css.scss [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  display: block;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 22px;
+  text-decoration: none;
+  font-size: 18px;
```
Comment (signalplus-dev) [inline app/assets/stylesheets/elements/_buttons.css.scss:36]: We should make all of our common font styles mixins and just include them when we need them. That way our typography stays consistent.

File: app/assets/stylesheets/elements/_panels.css.scss [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+.signal-panel {
+  margin-top: 28px;
+  color: $text-white;
+  margin-right: 20px;
+  display: inline-block;
+  width: 257px;
+  height: 280px;
+  border-radius: 6px;
+  background-color: #fafbfc;
+  box-shadow: 0 1px 2px 2px rgba(0, 0, 0, 0.2);
```
Comment (signalplus-dev) [inline app/assets/stylesheets/elements/_panels.css.scss]: I've seen this box-shadow twice. We should make it a variable.

File: app/assets/stylesheets/elements/_panels.css.scss [hunk 9]
```diff
@@ -0,0 +1,95 @@
+// Signal Panel styles
+
+.signal-panel {
+  margin-top: 28px;
+  color: $text-white;
+  margin-right: 20px;
+  display: inline-block;
+  width: 257px;
+  height: 280px;
+  border-radius: 6px;
```
Comment (signalplus-dev) [inline app/assets/stylesheets/elements/_panels.css.scss:10]: This `border-radius` value is probably very common. Make it a variable.

File: app/assets/stylesheets/elements/_panels.css.scss [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+  margin-bottom: 14px;
+}
+
+.panel-1 {
+  float: left;
+  color: $text-light;
+  background-color: $signal-turquoise;
+}
+
+.panel-2 {
```
Comment (signalplus-dev) [inline app/assets/stylesheets/elements/_panels.css.scss]: ??

File: app/assets/stylesheets/modules/_colors.css.scss [hunk 11]
```diff
@@ -0,0 +1,26 @@
+// Main Colors
```
Comment (signalplus-dev) [inline app/assets/stylesheets/modules/_colors.css.scss:1]: 👏

File: app/assets/stylesheets/modules/_base.css.scss [hunk 12]
```diff
@@ -0,0 +1,2 @@
+@import 'fonts';
```
Comment (signalplus-dev) [inline app/assets/stylesheets/modules/_base.css.scss:1]: 👏

File: app/assets/stylesheets/elements/_buttons.css.scss (hunk 5 above)
Comment (ghost) [inline app/assets/stylesheets/elements/_buttons.css.scss]: yea.. i just got lazy and copied it from chrome here.

**Pull request merged**